app/src/main/java/com/mapbox/mapboxsdk/plugins/testapp/activity/offline/OfflineRegionDetailActivity.kt

```diff
@@ -139,30 +140,30 @@ class OfflineRegionDetailActivity : AppCompatActivity(), OfflineDownloadChangeLi
         // update map
         mapView?.getMapAsync { mapboxMap ->
             // correct style
-            mapboxMap.setStyle(definition.styleURL)
-
-            // position map on top of offline region
-            val cameraPosition = OfflineUtils.getCameraPosition(definition)
-            mapboxMap.moveCamera(CameraUpdateFactory.newCameraPosition(cameraPosition))
-
-            // restrict camera movement
-            mapboxMap.setMinZoomPreference(definition.minZoom)
-            mapboxMap.setMaxZoomPreference(definition.maxZoom)
-            mapboxMap.setLatLngBoundsForCameraTarget(definition.bounds)
-        }
-
-        // update textview data
-        offlineRegion?.metadata?.let {
-            regionName.text = OfflineUtils.convertRegionName(it)
+            mapboxMap.setStyle(Style.Builder().fromUrl(definition.styleURL))
+            {
+                // position map on top of offline region
+                val cameraPosition = OfflineUtils.getCameraPosition(definition)
+                mapboxMap.moveCamera(CameraUpdateFactory.newCameraPosition(cameraPosition))
+
+                // restrict camera movement
+                mapboxMap.setMinZoomPreference(definition.minZoom)
+                mapboxMap.setMaxZoomPreference(definition.maxZoom)
+                mapboxMap.setLatLngBoundsForCameraTarget(definition.bounds)
+
+                // update textview data
+                offlineRegion?.metadata?.let {
+                    regionName.text = OfflineUtils.convertRegionName(it)
+                }
+                regionStyleUrl.text = definition.styleURL
+                regionLatLngBounds.text = definition.bounds.toString()
+                regionMinZoom.text = definition.minZoom.toString()
+                regionMaxZoom.text = definition.maxZoom.toString()
+                offlineRegion?.getStatus(offlineRegionStatusCallback)
+            }
```

Comment:
just as an FYI, we have a method on MapboxMap to make above a bit more easy:
`MapboxMap#setOfflineRegionDefinition(@NonNull OfflineRegionDefinition definition, @Nullable Style.OnStyleLoaded callback)`